src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -78,4 +79,10 @@ public ResponseEntity<Void> deleteEvent(@PathVariable UUID id) {
             return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR); // Unexpected error
         }
     }
+
+    // full path: /api/v1/events/{id}/users
+    @GetMapping("events/{id}")
```

Comment:
The mapping doesn't match the full path
```suggestion
    @GetMapping("events/{id}/users")
```